packages/editor/package.json

```diff
@@ -59,7 +59,7 @@
     "core-js": "^3.4.8",
     "express": "^4.17.1",
     "graphql": "^14.5.8",
-    "graphql-tag": "^2.10.1",
+    "graphql-tag": "^2.11.0",
```

Comment:
why the update?

Reply:
in the yarn.log a found 2 versions, i think it was the api which used `2.11.0` just tu use the same version on all pakcages

Reply:
Not part of this PR.